inst/include/cpp11/protect.hpp

```diff
@@ -195,23 +198,27 @@ constexpr struct protect safe = {};
 inline void check_user_interrupt() { safe[R_CheckUserInterrupt](); }
 
 template <typename... Args>
-void stop [[noreturn]] (const char* fmt, Args... args) {
-  safe.noreturn(Rf_errorcall)(R_NilValue, fmt, args...);
+void stop [[noreturn]] (const char* fmt_arg, Args... args) {
+  std::string msg = fmt::format(fmt_arg, args...);
```

Comment:
Wasn't aware of [this restriction](https://github.com/r-lib/cpp11/pull/209/checks?check_run_id=3128722782#step:12:76), adding perfect forwarding here should clean it up:
```suggestion
void stop [[noreturn]] (const char* fmt_arg, Args&&... args) {
  std::string msg = fmt::format(fmt_arg, std::forward<Args>(args)...);
```